pkg/resources/schema.go

```diff
@@ -90,7 +90,7 @@ func ValidateConnectionSchema() *schema.Schema {
 	}
 }
 
-func IdentifierSchema(elem, description string, required bool) *schema.Schema {
+func IdentifierSchema(elem, description string, required, forceNew bool) *schema.Schema {
```

Comment:
Having two boolean values in a function signature is pretty confusing, it seems easy to confuse the order. Normally I'd vote for using a struct here, but I realize that there are a lot of calls to this function and it's main purpose is to build a struct in the first place, so I'm not sure what the best solution is.

Reply:
Good idea! I just updated this to introduce a `IdentifierSchemaParams` struct.

Let me know what you think.